rafiki/container/docker_swarm.py

```diff
@@ -226,9 +227,8 @@ def retried_func(*args, **kwargs):
                 if no == RETRY_TIMES:
                     logger.info(f'Giving up on `{func} call...')
                     raise e
-                
+
```

Comment:
format the code to make the style consistent, tab=2 or 4? 
if you vscode, you can follow the instructions here https://github.com/apache/singa-doc/blob/master/docs-site/docs/contribute-code.md

Reply:
ok